src/types/Ctype.ts

```diff
@@ -1,10 +1,17 @@
 import * as sdk from '@kiltprotocol/sdk-js'
+import ICTypeMetadata, { IMetadata } from '@kiltprotocol/sdk-js/build/types/CTypeMetedata'
```

Comment:
please try to import everything from the index

Reply:
Yeah, I should check for this auto import issue more often